app/jobs/zipmaker_job.rb

```diff
@@ -33,4 +35,14 @@ def perform(druid, version, moab_replication_path) # rubocop:disable Lint/Unused
       PlexerJob.perform_later(druid, version, part_key, DruidVersionZipPart.new(zip, part_key).metadata)
     end
   end
+
+  # sleep for a configured amount of time, including a random additional delay to help ensure staggering.
+  #
+  # BUT WHY? to prevent duplication and possible explanation drift, see Robots::SdrRepo::PreservationIngest::UpdateCatalog#wait_as_needed
+  # for more detailed explanation.  the short version is, this helps alleviate Ceph MDS write/read sync and/or contention issues that
+  # we don't yet fully understand.
```

Comment:
I think the fuller explanation should be in a github issue (or even wiki).   

Staggering the zipmaker times ... again, we didn't used to need it -- if the real problem is the ceph access, then the stagger is not needed.  Over-engineered?

Reply:
yeah i was sort of cargo culting from what i've seen other delay mechanisms do.  happy to excise.